doc/en/components/charts/chart-features-animations.md

```diff
@@ -7,4 +7,61 @@ mentionedTypes: []
 
 # $PlatformShort$ Chart Animations
 
-<!-- category-chart-config-options.md -->
\ No newline at end of file
+Animations allows you to ease-in the series as it loads a new data source. The available animation differs depending on the type of series involved. For example, the column series animates by rising from the x-axis, a line series animates by drawing from the y-axis.
+
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/category-chart-line-chart-with-animations" 
+           alt="$PlatformShort$ Configuration Options Example" 
+           github-src="charts/category-chart/line-chart-with-animations">
+</code-view>
+
+<div class="divider--half"></div>
+
+## Transition In Animations
+
+By default, transition in animations are disabled and you can enable transition-in animations by setting the `IsTransitionInEnabled` option to “true”.
```

Comment:
replace "and you can enable transition-in animations"
with "but you can enable them"

Reply:
pending